docs/source/checks/vision/performance/plot_mean_average_precision_report.py

```diff
@@ -22,49 +22,43 @@
 
 """
 
-#%%
-# Imports
-# -------
-
-import numpy as np
-
-from deepchecks.vision.checks.performance import MeanAveragePrecisionReport
-from deepchecks.vision.datasets.detection import coco
-
-#%%
+# %%
 # Generate Data and Model
-# -----------------------
+# =================================
 # We generate a sample dataset of 128 images from the `COCO dataset <https://cocodataset.org/#home>`__,
 # and using the `YOLOv5 model <https://github.com/ultralytics/yolov5>`__.
 #
-# For the label formatter - our dataset returns exactly the accepted format, so our
-# formatting function is the simple `lambda x: x` function.
 
-yolo = coco.load_model(pretrained=True)
 
+from deepchecks.vision.checks.performance import MeanAveragePrecisionReport
+from deepchecks.vision.datasets.detection import coco
+
+yolo = coco.load_model(pretrained=True)
 test_ds = coco.load_dataset(train=False, object_type='VisionData')
 
-#%%
+# %%
 # Run the check
-# -------------
+# =================================
 
 check = MeanAveragePrecisionReport()
 result = check.run(test_ds, yolo)
 result
 
-#%%
+# %%
 # Observe the check’s output
 # --------------------------
-# The result value is a dataframe that has the average precision score per each area range and IoU.
+# The result value is a dataframe that has the Average Precision score for different bounding box sizes. We report the
+# Average Precision for different IoU thresholds: 0.5, 0.75 and a mean of AP values
+# for IoU thresholds between 0.5 and 0.9 (with jump size of 0.05).
 
 result.value
 
-#%%
+# %%
 # Define a condition 
 # ==================
-# We can define a condition that checks whether our model's average precision score is not less than
-# a given threshold
+# We can define a condition that checks whether our model's mean average precision score is not less than
+# a given threshold for all bounding box sizes.
 
-check = MeanAveragePrecisionReport().add_condition_test_average_precision_not_less_than(0.4)
+check = MeanAveragePrecisionReport().add_condition_test_mean_average_precision_not_less_than(0.4)
```

Comment:
Did you also change the condition in the check itself?

Reply:
No, just the condition we are displaying to the user in the doc.
The other one is less informative.